src/commonMain/kotlin/dev/inmo/krontab/StringParser.kt

```diff
@@ -39,32 +40,49 @@ typealias KrontabTemplate = String
  * * Days of month ranges can be found in [dayOfMonthRange]
  * * Months ranges can be found in [monthRange]
  * * Years ranges can be found in [yearRange] (in fact - any [Int])
+ * * Offset (timezone) ranges can be found in [offsetRange]
  *
  * Examples:
  *
  * * "0/5 * * * *" for every five seconds triggering
  * * "0/5,L * * * *" for every five seconds triggering and on 59 second
  * * "0/15 30 * * *" for every 15th seconds in a half of each hour
  * * "1 2 3 F,4,L 5" for triggering in near first second of second minute of third hour of fourth day of may
+ * * "1 2 3 F,4,L 5 60o" for triggering in near first second of second minute of third hour of fourth day of may with timezone UTC+01:00
  * * "1 2 3 F,4,L 5 2021" for triggering in near first second of second minute of third hour of fourth day of may of 2021st year
+ * * "1 2 3 F,4,L 5 2021 60o" for triggering in near first second of second minute of third hour of fourth day of may of 2021st year with timezone UTC+01:00
```

Comment:
As mentioned in [#14](https://github.com/InsanusMokrassar/krontab/issues/14) it's a bit _cumbersome_ to work with _offsets directly_. I think a _friendly approach_ would be to add a _thin convenience layer_ on top of _DateTimeTZ_ so that one could pass in ie _Europe/Berlin_ her instead of the actual offset. 

Anyways this is a start and i will check it out :)

Reply:
Currently I have added opportunity to get guaranteed `KronSchedulerTz`. In this case you will be able to use two ways:

1. Use your own way to save offset and pass it to methods like `createSimpleScheduler`
2. Use built-in opportunity to add offset via string-config parameter

Reply:
I think this is a good first version, i can work with offsets, but as an improvement i think it would be nice to offer a _thin layer on top of offsets_ as mentioned. Ie, a schedule would look like "1 2 3 F,4,L 5 Europe/Oslo" or something similiar. Then you convert Europe/Oslo internally to an offset. It's a bit cumbersome to work directly with offsets since you have to know the offset for the time zone you want to use.

Reply:
I am pretty sure that we will add this opportunity later:) for now we need to test and release version with offsets and weeks